### ISSUE TITLE
Swapping between clusters nukes credentials

### ISSUE THREAD

**Issue by kaihendry** [aws/amazon-ecs-cli#152]
ecs-cli version 0.4.3 (c6ea81e)

```
~/.ecs$ cat config
[ecs]
cluster                     = website
region                      = ap-southeast-1
aws_access_key_id           = MYID
aws_secret_access_key       = SECRET
compose-project-name-prefix = ecscompose-
compose-service-name-prefix = ecscompose-service-
cfn-stack-name-prefix       = amazon-ecs-cli-setup-
aws_profile                 = spuul
ame-prefix                  = amazon-ecs-cli-setup-

~/.ecs$ ecs-cli configure -c tv-app
INFO[0000] Saved ECS CLI configuration for cluster (tv-app)
~/.ecs$ cat config
[ecs]
cluster                     = tv-app
region                      =
aws_access_key_id           =
aws_secret_access_key       =
compose-project-name-prefix = ecscompose-
compose-service-name-prefix = ecscompose-service-
cfn-stack-name-prefix       = amazon-ecs-cli-setup-
aws_profile                 =
ame-prefix                  = amazon-ecs-cli-setup-

le                 = spuul
ame-prefix                  = amazon-ecs-cli-setup-
```

I think the workaround confusingly is `COMPOSE_PROJECT_NAME`, but I'm not 100% sure.

**Comment by pgeez** [comment]
I think this is related to #59 

👍

**Comment by joeboyscout04** [comment]
And the switching clusters also puts some junk at the bottom of the file, which causes "Error initializing: key-value delimiter not found"

**Comment by yinshiua** [comment]
related to https://github.com/aws/amazon-ecs-cli/issues/174

**Comment by yinshiua** [comment]
Added proposals to support multiple clusters configuration. Please give us some feedback. Thanks! https://github.com/aws/amazon-ecs-cli/issues/253

**Comment by yinshiua** [comment]
Released in v1.0.0
